onboarding-manager/internal/util/util.go

```diff
@@ -110,3 +119,29 @@ func PopulateInstanceIdleStatus(
 func PopulateCurrentOS(instance *computev1.InstanceResource, osResourceID string) {
 	instance.CurrentOs = &osv1.OperatingSystemResource{ResourceId: osResourceID}
 }
+
+func IsStandalone(instance *computev1.InstanceResource) (bool, error) {
```

Comment:
shall we not use Os at this point? I believe there will not be desired/current OS anymore